jina/types/arrays/memmap.py

```diff
@@ -17,6 +17,9 @@
 
 import numpy as np
 
+from ... import __windows__
+
+__windows__ = True
```

Comment:
@alaeddine-13 this is left from debugging?

Reply:
Oh sorry, will clean it